rfc-013-use-docker-to-deploy-and-run-applications-in-containers.md

```diff
@@ -0,0 +1,76 @@
+# Use Docker to deploy and run our applications in containers
+
+## Summary
+
+All of the applications we build are tested and run using Docker containers.
+
+This includes production environments as well as any staging environments and
+continuous integration services.
+
+The decision on whether we MUST use containers in development is not
+included in the scope of this RFC.
+
+## Problem
+
+To date there has been no formal decision to use or not use containers for
+hosting the applications we build. The result of this is a mixed ecosystem that
+is continuously growing.
+
+We currently host our work using numerous services and platforms:
+
+- Govpress
+- AWS
+- Azure
+- Heroku
+- Dalmatian
+- Government Platform as a Service (GPaaS)
+
+We cannot reduce the amount of services we use as doing so would reduce the
+flexibility we offer our clients. Clients sometimes require we use their tooling
+and platforms ensuring all their services can be managed centrally. This makes
+their services easier to operate as they move into live, it also allows dxw to
+more easily transition away from.
```

Comment:
Suggest rearranging the last bit of the sentence to "it also makes it easier for clients to transition away from dxw." I had some trouble parsing it as it is.